packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -134,119 +198,138 @@ export const PlanDetails = ({
                 ))}
               </ul>
 
-              <div className="plan-details-total">
-                {coupon && coupon.discountAmount && (
+              {loading ? (
+                <div className="loading">
+                  <LoadingSpinner />
+                </div>
+              ) : (
+                <div className="plan-details-total">
                   <div className="row-divider-grey-200 mb-4 pb-6">
-                    <div className="plan-details-total-inner">
-                      <Localized id="plan-details-list-price">
-                        <div>List Price</div>
-                      </Localized>
-
-                      <Localized
-                        id={`list-price`}
-                        attrs={{ title: true }}
-                        vars={{
-                          amount: getLocalizedCurrency(amount, currency),
-                          intervalCount: interval_count,
-                        }}
-                      >
-                        <div>
-                          {getLocalizedCurrencyString(amount, currency)}
-                        </div>
-                      </Localized>
-                    </div>
+                    {subTotal && (
+                      <div className="plan-details-total-inner">
+                        <Localized id="plan-details-list-price">
+                          <div>List Price</div>
+                        </Localized>
+                        <Localized
+                          id={`list-price`}
```

Comment:
Ignorant React question, but is this different from just vanilla `id="list-price"`?

Reply:
This was another change that I think only got included because of the formatter/linter running but I think in this question you'd be exactly right in saying that its not different. The `{}` are for evaluating expressions and the `` are usually for a string interpolation; this is just a regular string though. My guess is that this is in there because of some copy pasta where the line item was copied and pasted and the IDs were updated.